bindings/python/setup.py

```diff
@@ -112,6 +112,9 @@ def target_specific():
 		extra_link = flags.parse(ldflags)
 		extra_compile = flags.parse(extra_cmd)
 
+		if 'g++' in os.environ['CXX'] or 'clang' in os.environ['CXX']:
+			os.environ["CC"] = os.environ['CXX']
```

Comment:
does python's distutils always use $CC to build? even c++? I don't understand why this is necessary.